projects/solidity/build.sh

```diff
@@ -66,7 +66,7 @@ rm -f $OUT/*.zip
 for dir in $SRC/solidity-fuzzing-corpus/*;
 do
 	name=$(basename $dir)
-	zip -ujq $OUT/$name.zip $dir/* &>/dev/null
+	zip -rjq $OUT/$name $dir
```

Comment:
Are you sure you need `-j`? We read the corpus directories recursively, so the flat structure is not necessary.